### PR TITLE
Created contentTile. Changed mainContent.

### PR DESCRIPTION
**Pull request by magmat88**
Created contentTile component. Changed mainContent component to make
this component use contentTile.

### CODE REVIEW

File: src/app/components/blogs/blogs.component.scss [hunk 1]
```diff
@@ -2,6 +2,6 @@
 @import '../../styles/placeholders/placeholders.scss';
 @import '../../styles/variables/variables.scss';
 
-.blogs-container {
-    @extend %columnSpaceBetween;
-}
\ No newline at end of file
+// .blogs-container {
```
Comment (earendillo) [inline src/app/components/blogs/blogs.component.scss:5]: remove commented code

File: src/app/components/contentItem/contentTile.component.scss [hunk 2]
```diff
@@ -0,0 +1,114 @@
+@import "../../styles/mixins/mixins.scss";
+@import "../../styles/placeholders/placeholders.scss";
+@import "../../styles/variables/variables.scss";
+
+.content-tile {
+    @extend %marginStandard;
+    @extend %paddingZero;
+    @extend %paletteLight;
+    @extend %rowSpaceBetween;
+    border: 1px solid green;
```
Comment (earendillo) [inline src/app/components/contentItem/contentTile.component.scss:10]: remove this border

File: src/app/components/contentItem/contentTile.component.scss [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+    @include forLargeScreens($breakpoint-medium) {
+        height: 150px;
+        width: 60%;
+    }
+}
+
+.content-tile-img {
+    height: 280px;
+    margin: 10px;
+    margin-right: 0px;
```
Comment (earendillo) [inline src/app/components/contentItem/contentTile.component.scss:39]: set margin in single  line

Comment (earendillo) [reply src/app/components/contentItem/contentTile.component.scss:39]: margin: 10px 0 10px 10px

File: src/app/components/mainContent/mainContent.component.scss [hunk 4]
```diff
@@ -7,39 +7,41 @@
     @extend %paletteMedium;
 }
 
-.articles-and-blogs-container {
-    @extend %columnCentered;
-
+// .articles-and-blogs-container {
```
Comment (earendillo) [inline src/app/components/mainContent/mainContent.component.scss:10]: remove commented code

File: src/app/components/mainContent/mainContent.component.scss [hunk 5]
```diff
@@ -7,39 +7,41 @@
...
+.reports-container {
+    @extend %columnCentered;
+    height: 300%;
     width: 100%;
 
     @include forLargeScreens($breakpoint-medium) {
-        width: 50%;
+        flex-direction: row;
+        width: 100%;
+        height: 33.333333%;
```
Comment (earendillo) [inline src/app/components/mainContent/mainContent.component.scss:27]: why such value?

File: src/app/components/mainContent/mainContent.component.scss [hunk 6]
```diff
@@ -7,39 +7,41 @@
...
     }
 }
 
-img {
-    @extend %paddingStandard;
-    @include setImageDisplay(auto, 150px);
-}
\ No newline at end of file
+// .article-item,
```
Comment (earendillo) [inline src/app/components/mainContent/mainContent.component.scss:31]: remove commented code

**Review by earendillo — CHANGES_REQUESTED** [review]

File: src/app/components/blogs/blogs.component.scss (hunk 1 above)
Comment (magmat88) [reply src/app/components/blogs/blogs.component.scss:5]: remove commented code and unnecessary imports

**Review by earendillo — APPROVED** [review]

**Pull request merged**